Microsoft.Toolkit/Helpers/ObjectStorage/IFileStorageHelper.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Helpers
+{
+    /// <summary>
+    /// Service interface used to store data in files and folders.
+    /// </summary>
+    public interface IFileStorageHelper
+    {
+        /// <summary>
+        /// Determines whether a file already exists.
+        /// </summary>
+        /// <param name="filePath">Key of the file (that contains object).</param>
+        /// <returns>True if a value exists.</returns>
+        Task<bool> FileExistsAsync(string filePath);
```

Comment:
I like `ItemExistsAsync`. Somehow I forgot a `FolderExistsAsync` method, so this is great!

I thought about how I might transition folks between them, but I think the clean change over is the best approach. Otherwise I end up making weird concessions in the new code to accommodate the old interface.

Reply:
Yeah, makes sense. We could have an extra flag potentially for checking if it's specifically a file vs. folder or something, but not sure if that's useful.

Mentioned in another comment an approach for handling breaking changes so we can still have a smooth upgrade path for 7.1 for folks, as I do really like that they can just swap `LocalObjectStorageHelper` for this new uber implementation.

Reply:
I can implement `FileExistsAsync` in the new `ApplicationDataStorageHelper` as well. That way both will `FileExistsAsync` and `ItemExistsAsync` will be available to hot swappers, but won't pollute the new interface with an extra method. :)